.github/workflows/ci.yml

```diff
@@ -63,6 +63,28 @@ jobs:
       - run: pre-commit run --all-files
 
 
+  test:
+    name: Testing
+    runs-on: ubuntu-latest
+    steps:
+
+      - uses: actions/checkout@v4
+
+      - uses: actions/setup-python@v5
+        with:
+          python-version-file: ./backend/.python-version
+
+      - name: Install uv
+        uses: astral-sh/setup-uv@v5
+        with:
+          enable-cache: true
+          cache-dependency-glob: ./backend/uv.lock
+
+      - name: Python unit testing
+        working-directory: ./backend/
+        run: uv run pytest
```

Comment:
A question, will we eventually want to run a percentage check for unit test as well? Or do you see you see this as something we might add later on. I see it as the latter but would like to know your opinion.

Reply:
Yeah, eventually.  Right now we're at some super low percentage.  Sadness.  But yes, something later on.